src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -139,7 +66,10 @@ private boolean hasEventId(CaseTypeDefinition caseTypeDefinition, String eventId
     public void validateData(Map<String, JsonNode> data,
                              CaseTypeDefinition caseTypeDefinition,
                              final CaseDataContent content) {
-        caseTypeService.validateData(data, caseTypeDefinition);
-        validateOrganisationPolicy(caseTypeDefinition.getId(), content);
```

Comment:
I think this could have been a little bug here. We were passing to `validateOrganisationPolicy` `content`, but that might not contain the most up to date case data. We should be using `data` instead

Reply:
Given the above, if we pass the whole `CaseDataContent` to the ValidationContext, we risk devs using `caseDataContent.getData()` rather than the more up to date `data`. 
Therefore I think it would be better to do this:
```
createValidationContext(Map<String, JsonNode> data, CaseTypeDefinition caseTypeDefinition, String eventId)
```
where `data` will contain 
`Map<String, JsonNode> data`

Reply:
DONE